eo-maven-plugin/src/main/java/org/eolang/maven/optimization/OptSpy.java

```diff
@@ -67,8 +67,12 @@ public OptSpy(final Train<Shift> trn, final Path target) {
 
     @Override
     public XML apply(final XML xml) {
-        final Place place = new Place(xml.xpath("/program/@name").get(0));
-        final Path dir = place.make(this.target, "");
+        final Path dir = new Place(
+            xml
+                .xpath("/program/@name")
+                .get(0)
+        )
+            .make(this.target, "");
```

Comment:
@Yanich96 Do you need a separate line for this statement? 
Can we just write the following in one line:
```java
 final Path dir = new Place(xml .xpath("/program/@name").get(0)).make(this.target, "");
```
or in two lines:
```java
 final Path dir = new Place(xml .xpath("/program/@name").get(0))
   .make(this.target, "");
```